apport: report to the "pantheon-dock" dummy project
data: add NotShowIn=Pantheon; to desktop-file
dockpreferences: update internal defaults for "Pantheon"

=== modified file 'data/apport/plank-crashdb.conf'
--- old/data/apport/plank-crashdb.conf	2012-03-09 21:45:40 +0000
+++ new/data/apport/plank-crashdb.conf	2013-10-13 19:29:22 +0000
@@ -1,5 +1,5 @@
 plank = {
 	'impl' : 'launchpad',
-	'project' : 'plank',
+	'project' : 'pantheon-dock',
 	'bug_pattern_base' : None,
 }

=== modified file 'data/plank.desktop.in'
--- old/data/plank.desktop.in	2012-10-09 16:37:49 +0000
+++ new/data/plank.desktop.in	2013-10-13 19:29:22 +0000
@@ -8,3 +8,4 @@
 Icon=plank
 Terminal=false
 NoDisplay=false
+NotShowIn=Pantheon;

=== modified file 'lib/DockPreferences.vala'
--- old/lib/DockPreferences.vala	2013-10-07 15:36:50 +0000
+++ new/lib/DockPreferences.vala	2013-10-13 19:30:02 +0000
@@ -108,13 +108,13 @@
 			
 			CurrentWorkspaceOnly = false;
 			IconSize = 48;
-			HideMode = HideType.INTELLIGENT;
+			HideMode = HideType.DODGE_MAXIMIZED;
 			UnhideDelay = 0;
 			Monitor = -1;
 			DockItems = "";
 			Position = Gtk.PositionType.BOTTOM;
 			Offset = 0;
-			Theme = Plank.Drawing.Theme.DEFAULT_NAME;
+			Theme = Plank.Drawing.Theme.GTK_THEME_NAME;
 			Alignment = Gtk.Align.CENTER;
 			ItemsAlignment = Gtk.Align.CENTER;
 			LockItems = false;

=== modified file 'lib/Drawing/Theme.vala'
--- old/lib/Drawing/Theme.vala	2014-12-16 09:11:36 +0000
+++ new/lib/Drawing/Theme.vala	2014-12-24 08:53:52 +0000
@@ -387,9 +387,9 @@
 				&& folder.query_file_type (FileQueryInfoFlags.NONE, null) == FileType.DIRECTORY)
 				return folder;
 			
-			warning ("%s not found, falling back to %s.", name, DEFAULT_NAME);
+			warning ("%s not found, falling back to %s or %s.", name, GTK_THEME_NAME, DEFAULT_NAME);
 			
-			return get_default_theme_folder ();
+			return get_gtk_theme_folder () ?? get_default_theme_folder ();
 		}
 		
 		static File? get_default_theme_folder ()

